processes

```diff
@@ -1,4 +1,4 @@
-screening
-pairing
-pretty_set
-constructor
\ No newline at end of file
+screening bulk_pit
+pairing bulk_pit
+pretty_set bulk_pit
+constructor bulk_pit
```

Comment:
The bulk_pit should go in the task list not processes

Reply:
Changed back to default values